modules/apps/object/object-storage-salesforce/src/main/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/SalesforceObjectEntryManagerImpl.java

```diff
@@ -359,6 +367,24 @@ private ObjectField _getObjectFieldByName(
 		return null;
 	}
 
+	private ArrayList<String> _getObjectFieldExternalReferenceCodes(
+		List<ObjectField> objectFields) {
+
+		ArrayList<String> objectFieldExternalReferenceCodes = new ArrayList<>(
+			Arrays.asList("OwnerId", "CreatedDate", "LastModifiedDate", "Id"));
+
+		for (ObjectField objectField : objectFields) {
+			String externalReferenceCode =
+				objectField.getExternalReferenceCode();
+
+			if (!externalReferenceCode.matches(_UUID_REGEX_PATTERN)) {
+				objectFieldExternalReferenceCodes.add(externalReferenceCode);
+			}
+		}
+
+		return objectFieldExternalReferenceCodes;
+	}
```

Comment:
Hey @jairmedeiros since we're using this only in one place we can move this logic to the place we're using.

Thanks!